src/App.tsx

```diff
@@ -1,8 +1,17 @@
-import React from 'react';
+import React, { useState } from 'react';
+import cn from 'classnames';
 import 'bulma/css/bulma.css';
 import './App.scss';
 
-export const goodsFromServer = [
+type Good = string;
```

Comment:
Do we realy need custom type here?

Reply:
@kaavka This type will allow you to more clearly specify the typification for the array:
```js
const goods: Good[] = ...
```